homeassistant/components/victron_gx_mqtt/strings.json

```
@@ -0,0 +1,1335 @@
+        "name": "Battery minimum voltage cell ID"
+      },
+      "battery_nr_modules_blocking_charge": {
+        "name": "Battery number of modules blocking charge"
+      },
+      "battery_nr_modules_blocking_discharge": {
+        "name": "Battery number of modules blocking discharge"
+      },
+      "battery_nr_modules_offline": {
+        "name": "Battery number of modules offline"
+      },
+      "battery_nr_modules_online": {
+        "name": "Battery number of modules online"
+      },
+      "battery_power": {
+        "name": "Battery power"
+      },
+      "battery_soc": {
+        "name": "Battery charge"
+      },
+      "battery_soh": {
+        "name": "Battery state of health"
+      },
+      "battery_temperature": {
+        "name": "Battery temperature"
+      },
+      "battery_time_to_go": {
+        "name": "Battery time to go"
+      },
+      "battery_voltage": {
+        "name": "DC Bus voltage"
+      },
+      "charge_mode": {
+        "name": "Battery charge mode"
+      },
+      "dcload_current": {
+        "name": "DC load current"
+      },
+      "dcload_power": {
+        "name": "DC load power"
+      },
+      "dcload_voltage": {
+        "name": "DC load voltage"
+      },
+      "dcsystem_aux_voltage": {
+        "name": "DC System auxiliary voltage"
+      },
+      "dcsystem_current": {
+        "name": "DC System current"
+      },
+      "dcsystem_power": {
+        "name": "DC System power"
+      },
+      "dcsystem_voltage": {
+        "name": "DC System voltage"
+      },
+      "digitalinput_alarm": {
+        "name": "Digital input alarm"
+      },
+      "digitalinput_input_state_raw": {
+        "name": "Digital input raw state"
+      },
+      "digitalinput_state": {
+        "name": "Digital input state"
+      },
+      "digitalinput_type": {
+        "name": "Digital input type"
+      },
+      "evcharger_current": {
+        "name": "EV charger current"
+      },
+      "evcharger_mode": {
+        "name": "EV charger mode"
+      },
+      "evcharger_power_phase": {
+        "name": "EV charger power {phase}"
+      },
+      "evcharger_set_current": {
+        "name": "EV charger set current"
+      },
+      "evcharger_status": {
+        "name": "EV charger status"
+      },
+      "generator_gen_id_cool_down_timer": {
+        "name": "Generator cool down timer"
+      },
+      "generator_gen_id_qh_start_on_soc": {
+        "name": "Generator QH start on SOC"
+      },
+      "generator_gen_id_qh_start_on_voltage": {
+        "name": "Generator QH start on voltage"
+      },
+      "generator_gen_id_qh_stop_on_soc": {
+        "name": "Generator QH stop on SOC"
+      },
+      "generator_gen_id_qh_stop_on_voltage": {
+        "name": "Generator QH stop on voltage"
+      },
+      "generator_gen_id_service_interval": {
+        "name": "Generator service interval"
+      },
+      "generator_gen_id_shut_down_timer": {
+        "name": "Generator shut down timer"
```

"shutdown" is a noun here, otherwise the generator shuts down the timer …

```suggestion
        "name": "Generator shutdown timer"
```